docs/README.md

```diff
@@ -1,106 +1,109 @@
-- Offers compatibility with multiple media transport protocols (SMPTE ST 2110, RTSP, ...).
+## 1. Media Communications Mesh:
+-	Framework Introduction (MCM): is a framework designed for low-latency media transport. It caters specifically to Edge, Edge-to-Cloud, and Cloud environments, connecting media microservices.
+-	Purpose: Its main role is to establish a secure media data plane for inter-microservices communication. This ensures media can be reliably transmitted across different cloud environments.
+-	Technological Basis: The Media Transport Library (MTL) helps form the media data plane, and a control-plane communications infrastructure adds support for media control protocols.
 
-### MemIF
+✅ Key point: The main goal is to enable secure, fast, and standardized media communication between microservices, regardless of the environment (Edge or Cloud).
 
-The Media Proxy leverages the [MemIF](https://s3-docs.fd.io/vpp/24.02/interfacing/libmemif/index.html) protocol for implementing shared memory buffer communications.
+## 2. Media Proxy Introduction:
+-	Core Functionality: The Media Proxy handles the routing and forwarding of media data, specifically audio and video streams, between mesh nodes.
+-	Low Latency & Efficiency: The focus is on ensuring low-latency and efficient usage of system resources, which is critical for real-time media applications.
+-	Service Mesh Role: It acts as a Data Plane component within a larger service mesh for media applications, utilizing Shared Memory APIs to abstract the complexities involved in media transport.
 
-## MCM SDK
+✅ Key point: Media Proxy handles the data transport layer of media streams, focusing on real-time efficiency and ensuring media routing between microservices in the mesh.
+✅ More information about the Media Proxy component can be found in the [media-proxy](media-proxy) subdirectory.
 
-The MCM SDK library is designed for microservice applications to offload media transport functionalities. By abstracting network transport functions for media data and wrapping the "libmemif" APIs, MCM SDK facilitates communication with "Media Proxy".
+## 3. SDK and FFmpeg plugins:
+-	Key Features: Zero Memory Copy, uses a zero-copy memory techniques for ultra-low-latency media transfers between containers. This helps avoid data duplication, reducing time and resource overhead.
+-	Media Stream Compatibility: Supports compressed (like JPEG XS) and RAW uncompressed media formats, ensuring flexibility in how media is transported.
+-	Multiple Protocols: Works with protocols such as SMPTE ST 2110 and RTSP, enhancing its versatility in different media service setups.
 
-Detailed information about MCM SDK can be found in [sdk](sdk) directory.
+✅ Key point: The system is optimized for zero-copy transmission with wide protocol compatibility, ensuring it can handle both compressed and uncompressed media streams efficiently.
+✅ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
```

Comment:
line breaks do not work here (before "v" signs).

Reply:
fixed